engine/phase.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+class Phase(Enum):
+    """
+    An enumeration of different robot phases.
+    """
+    SETUP = 1
+    TRAVERSE = 2
+    AVOID_OBSTACLE = 3
+    RETURN = 4
+    DOCKING = 5
+    COMPLETE = 6
+    FAULT = 7
```

Comment:
might not be relevant to this pr, but I remember we might be having a phase for manual control?

Reply:
yus not really relevant bc phase was just moved into a separate file for reducing robot clutter but good point I will add it to our list of other things we should figure out